addons/website_sale/models/product.py

```diff
@@ -17,7 +18,23 @@ class ProductPricelist(models.Model):
     _inherit = "product.pricelist"
 
     def _default_website(self):
-        return self.env['website'].search([], limit=1)
+        """ Find the first company's website, if there is one. """
+        company_id = self.env.user.company_id.id
+        if self.pool._init:
```

Comment:
so all test will be faked?

Reply:
Correct me if I am wrong but from what I have tried, `self.pool._init` is always `False` while tests are being runned, both at module install and after install, and with `--test-enable` or `--test-file` (+[1615869d](https://github.com/odoo-dev/odoo/blame/1615869d72c02896c84562c56cb72fcab2b78d6b/addons/mail/models/res_partner.py#L173).)